src/script/common/c_converter.cpp

```diff
@@ -457,12 +456,22 @@ size_t read_stringlist(lua_State *L, int index, std::vector<std::string> *result
 	Table field getters
 */
 
+bool check_field_or_nil(lua_State *L, int type, const char *fieldname)
```

Comment:
Should have an `index` argument even if it's always `-1`

Reply:
added